libs/ngx-dsfr/radio/src/lib/radio.component.ts

```diff
@@ -69,10 +71,28 @@ export class DsfrRadioComponent implements ControlValueAccessor, OnInit {
     if (this.items.length === 0) {
       throw EMPTY_ITEMS_ERROR;
     }
-    this.ngControl.control?.valueChanges.subscribe(value => {
-      if (this._value === value) return;
-      this.writeValue(value);
-    });
+    this.setFieldSetClasses();
+    this.setAriaLabelledBy();
```

Comment:
Are we sure this is the only moment when we need to compute all that ?
They depend on `hasFailed` and `hasSucceeded` : won't they change during the lifecycle of this Component ?